index.js

```diff
@@ -23,68 +26,72 @@ function getRandomNumber(min, max) {
 }
 
 function checkGuess() {
-  // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
 
   hideAllMessages();
 
-  if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+  if (guess < guessInput.min) {
+    wayTooLow.style.display = "block";
+  } else if (guess > guessInput.max) {
+    wayTooHigh.style.display = "block";
+  } else if (guess === targetNumber) {
+    numberOfGuessesMessage.style.display = "";
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
-    correctMessage.style.display = '';
-
+    correctMessage.style.display = "";
     submitButton.disabled = true;
     guessInput.disabled = true;
-  }
 
-  if (guess !== targetNumber) {
+    if (attempts === maxNumberOfAttempts) {
```

Comment:
Because of another bug, and because this check is only for `===` instead of `>=`, I was able to get my guesses to go into negative numbers =D

<img width="271" alt="Screen Shot 2023-09-16 at 8 27 14 PM" src="https://github.com/CristianMoran1/intro-to-programming-section-5-1/assets/12771651/3c9c916f-5a9e-40ea-99f5-bd8565735487">